lib/active_model_serializers/key_transform.rb

```diff
@@ -0,0 +1,43 @@
+module ActiveModelSerializers
+  module KeyTransform
```

Comment:
perhaps `module_function` here to allow public calling on the module but private methods on inclusion. I like me some utility function modules